eos-paygd/main.c

```diff
@@ -35,11 +35,14 @@
 #include <sys/un.h>
 #include <libeos-payg/efi.h>
 
+#define POWEROFF_TIMEOUT_ERROR 20
```

Comment:
Perhaps call it `POWEROFF_TIMEOUT_ON_ERROR_MINUTES`? Suffixing with `_ERROR` makes it look like this is an error or exit code. Suffixing with `_MINUTES` makes it clearer what the units are (as time is equally often measured in seconds or microseconds).

Reply:
Interesting, I usually expect error codes to be prefixed with `ERROR` instead of suffixed. I don't care too much, so if it makes it easier for others to read I'll take this. But I actually went with `TIMEOUT_POWEROFF_NO_CREDIT_MINUTES`, which seems more organized.